src/store/reducer/LoginReducer.ts

```diff
@@ -0,0 +1,25 @@
+import { LOGIN_FAIL, LOGIN_SUCCESS, GET_REQ_ERR } from "store/types";
+const initialState = {
+  loading: false,
+  success: false,
+  error: false,
+  user: null,
+};
+
+const reducer = (state = initialState, action: any) => {
+  const newState = { ...state };
+  switch (action.type) {
+    case LOGIN_SUCCESS:
+      newState.loading = false;
+      newState.success = true;
```

Comment:
Make a third reducer App and move below in that reducer.
- loading
- success
- error 

Then both your login and post actions can use this reducer keys. and it will be shared.